app/views/users/index.html.erb

```diff
@@ -4,7 +4,17 @@
     <title>List of Users</title>
   </head>
   <body>
-    <h1>This is the list of Users page</h1>
-    <p>parameters: <%= params %></p>
-</body>
+    <ul class="user-list">
+      <% @users.each do |user| %>
+        <li class="user-item">
+          <img src="<%= user.photo %>" alt="User Photo" class="user-photo">
+          <div class="user-info">
+            <h2><%= user.name %></h2>
+            <p>Number of Posts: <%= user.posts_counter %></p>
+          </div>
+          <a class= 'link-see-user-profile link-btn' href="<%= user_path(user) %>">See User Profile</a>
+        </li>
+      <% end %>
+    </ul>
```

Comment:
- [ ] It is required that clicking on a user should redirect to the user's show page. Kindly make each user a link to the user's show page. I understand `See User Profile` serves that purpose, but according to the project requirement, every user should be made a link to their respective show page. Kindly do just that.

Reply:
ok. I'll do it. thank you 👍🏽